### PR TITLE
chore: Add deployment for MaterialX sample web app

### PR DESCRIPTION
**Pull request by kohakukun**

### CODE REVIEW

**Comment by kohakukun** [comment]
Build failing at the moment because pr is done from a different fork. 
It worked when creating a branch directly: https://github.com/autodesk-forks/MaterialX/runs/3045495332

Which create the last commit in https://github.com/autodesk-forks/MaterialX/tree/gh-pages

File: source/JsMaterialX/JsMaterialXView/src/index.js [hunk 1]
```diff
@@ -222,6 +233,8 @@ function onWindowResize() {
 
 function animate() {  
     requestAnimationFrame(animate);
+    const currentTime = Date.now();
```
Comment (muenstc) [inline source/JsMaterialX/JsMaterialXView/src/index.js:236]: What is this for?

File: source/JsMaterialX/JsMaterialXView/webpack.config.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
 const path = require('path');
+const fs = require('fs');
 const CopyPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+const stdSurfaceMaterial = "../../../resources/Materials/Examples/StandardSurface";
```
Comment (muenstc) [inline source/JsMaterialX/JsMaterialXView/webpack.config.js:6]: Nit: `stdSurfaceMaterials`

**Review by muenstc — COMMENTED** [review]
LGTM

File: source/JsMaterialX/JsMaterialXView/src/index.js (hunk 1 above)
Comment (frericp) [inline source/JsMaterialX/JsMaterialXView/src/index.js:236]: This doesn't seem to be used.

File: source/JsMaterialX/JsMaterialXView/src/index.js [hunk 3]
```diff
@@ -186,7 +197,7 @@ function init() {
         });
         obj.traverse((child) => {
             if (child.isMesh) {
-              generateTangents(child.geometry);
+              child.geometry.computeTangents();
```
Comment (frericp) [inline source/JsMaterialX/JsMaterialXView/src/index.js:200]: Oh, why did we do it ourselves before?

File: source/JsMaterialX/JsMaterialXView/webpack.config.js [hunk 4]
```diff
@@ -9,18 +17,26 @@ module.exports = {
...
         { 
           context: "../../../resources/Images",
           from: "*.jpg", 
           to: "Images",
         },
+        { from: "./public", to: 'public' },
         { from: "../../../resources/Images/greysphere_calibration.png", to: "Images" },
-        { from: "../../../resources/Geometry/shaderball.obj",  to: "Geometry"},
+        { from: "../../../resources/Geometry/shaderball.glb",  to: "Geometry"},
+        { from: "node_modules/three/examples/js/libs/draco",  to: "draco"},
```
Comment (frericp) [inline source/JsMaterialX/JsMaterialXView/webpack.config.js:36]: I couldn't find where this is used. Do we need to copy it?

**Review by frericp — COMMENTED** [review]
A few minor comments. Looks good otherwise.

File: source/JsMaterialX/JsMaterialXView/src/index.js (hunk 3 above)
Comment (muenstc) [reply source/JsMaterialX/JsMaterialXView/src/index.js:200]: that function only works if there is an index buffer

File: source/JsMaterialX/JsMaterialXView/webpack.config.js (hunk 4 above)
Comment (kohakukun) [reply source/JsMaterialX/JsMaterialXView/webpack.config.js:36]: yeah, sadly. It needs to load the wasm files. Actually we had a similar issue with Karma and we were wondering how other libraries do :/

Comment (kohakukun) [reply source/JsMaterialX/JsMaterialXView/webpack.config.js:36]: This is used in the index.js `dracoLoader.setDecoderPath( '/draco/' );`

**Review by muenstc — APPROVED** [review]

File: source/JsMaterialX/JsMaterialXView/src/index.js (hunk 3 above)
Comment (frericp) [reply source/JsMaterialX/JsMaterialXView/src/index.js:200]: Ok, so we can now assume that there's always such a buffer with the new file loader?

**Pull request merged**